src/components/ApplicationMenu/ApplicationMenu.messages.js

```diff
@@ -0,0 +1,357 @@
+// @flow
+import { defineMessages } from 'react-intl';
+
+const messages = {
+  file: {
+    label: {
+      id: 'applicationMenu.file.label',
+      defaultMessage: '&File',
```

Comment:
Is the accelerator working? I wasn't sure but it's working as expected. By pressing alt key on Windows it will display the acceleator by unterlining the Letter F in the menu bar.